static/js/src/advantage/subscribe/checkout/components/BuyButton/BuyButton.tsx

```diff
@@ -271,19 +271,22 @@ const BuyButton = ({ setError, quantity, product, action }: Props) => {
       request.onreadystatechange = () => {
         if (request.readyState === 4) {
           localStorage.removeItem("shop-checkout-data");
-          if (!window.loginSession) {
+          if (product.marketplace == "canonical-cube") {
+            console.log("Reaches here 1");
```

Comment:
Maybe remove this?

Reply:
Please do indeed remove the console.log. And fix linting.